pkg/controller/clusterprovision/installlogmonitor_test.go

```diff
@@ -265,43 +267,15 @@ func TestParseInstallLog(t *testing.T) {
 			expectedReason: "DNSAlreadyExists",
 		},
 		{
-			name: "GCP compute quota",
-			log:  pointer.StringPtr(gcpCPUQuotaLog),
-			existing: []runtime.Object{&corev1.ConfigMap{
-				ObjectMeta: metav1.ObjectMeta{
-					Name:      regexConfigMapName,
-					Namespace: constants.DefaultHiveNamespace,
-				},
-				Data: map[string]string{
-					"regexes": `
-- name: GCPComputeQuota
-  searchRegexStrings:
-  - "compute\\.googleapis\\.com/cpus is not available in [a-z0-9-]* because the required number of resources \\([0-9]*\\) is more than remaining quota"
-  installFailingReason: GCPComputeQuotaExceeded
-  installFailingMessage: GCP CPUs quota exceeded
-`,
-				},
-			}},
+			name:           "GCP compute quota",
+			log:            pointer.StringPtr(gcpCPUQuotaLog),
+			existing:       []runtime.Object{buildRegexConfigMap()},
```

Comment:
This is latent, so could be handled in a separate PR, but is there any reason to build this on every test case? It's read-only, right?

Reply:
There are some test cases that want a missing config or a malformed regex so that's why I have it per test case atm.

Reply:
Could split them in a separate PR

Reply:
> some test cases that want a missing config or a malformed regex

So there are two different issues here.
1. This comment was about all test cases where `existing` is simply using `buildRegexConfigMap()`. All of those test cases should be able to reuse a common variable built once at the top of the function. That's the bit I'm saying was latent and could be handled in a separate PR.
2. Some of the test cases don't use `buildRegexConfigMap()` but instead use inlined configmaps. Are all of those specifically testing edge cases with missing/bad configmaps?